static/src/js/components/CollectionSelector/__tests__/CollectionSelector.test.jsx

```diff
@@ -270,7 +270,7 @@ describe('CollectionSelector', () => {
 
       await user.type(searchField, 'C')
 
-      expect(await screen.findByText('Showing 2 items')).toBeInTheDocument()
+      expect(await screen.findByText('Showing 1 items')).toBeInTheDocument()
```

Comment:
Why did this change to 1 again?

Reply:
I'm not sure, but the mock only has 1 collection as the result.   So it is correct.